tests/components/automation/test_event.py

```diff
@@ -7,10 +7,11 @@
 from tests.common import get_test_home_assistant
 
 
+# pylint: disable=invalid-name
```

Comment:
It's unfortunate this can't be set for the complete test folder. Maybe we should just add it top of file for all tests?